CONTRIBUTING.md

```diff
@@ -9,6 +9,46 @@ Set up your environment by following the instructions at
 https://pytorch.org/executorch/stable/getting-started-setup.html to clone
 the repo and install the necessary requirements.
 
+Refer to this [document](https://pytorch.org/executorch/main/using-executorch-building-from-source.html) to build ExecuTorch from source.
+
+### Dev Setup for Android
+For Android, please refer to the [Android documentation](docs/source/using-executorch-android.md).
+
+### Dev Setup for Apple
+For Apple, please refer to the [iOS documentation](docs/source/using-executorch-ios.md).
+&nbsp;
+
+## Codebase structure
```

Comment:
In the top-level README.md file, can you the repo structure, and just point to this section instead? Let's not have two copies. 

https://github.com/pytorch/executorch/blob/main/README.md?plain=1#L66

Reply:
ok sounds good I'll merge